tests/integration/test_background_resumability.py

```
@@ -0,0 +1,458 @@
+        config=StepConfig(max_retries=0, execution_mode="background"),
+    )
+    pipeline = Pipeline.from_step(bg_step)
+
+    async with Flujo(
+        pipeline,
+        context_model=BgContext,
+        state_backend=backend,
+        hooks=[recorder],
+    ) as runner:
+        executor = runner.backend._executor  # type: ignore[attr-defined]
+        executor._get_background_quota = lambda parent_quota=None: quota_holder  # type: ignore[assignment]
```

Ruff (0.14.6)

Unused lambda argument: `parent_quota`

(ARG005)